notebooks/Emukit-basic-use-of-the-library.ipynb

```diff
@@ -0,0 +1,349 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Basic use of the Emukit\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Overview"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# General imports and parameters of figures should be loaded at the beginning of the overview\n",
+    "\n",
+    "### General imports\n",
```

Comment:
While we are at it, could be unify comments, please? I'd vote for a simple # only.

Reply:
It is a subsection rather than a comment. I have now added more comments to the notebook to clarify the steps.

Reply:
Keeping one # in the comments